src/app/shared/stache/modules/video/video.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="stache-video">
+  <iframe class="stache-video-embed" [src]="src" frameborder="0" allowfullscreen></iframe>
```

Comment:
We can come back to this idea later, just wanted to get our minds thinking about it. I believe SKY UX is planning on building an iframe component.